src/webgl/p5.Geometry.js

```diff
@@ -1398,7 +1399,7 @@ class Geometry {
       if (dirOK) {
         this._addSegment(begin, end, fromColor, toColor, dir);
       }
-
+      if(!this.renderer._simpleLines){
```

Comment:
Since `p5.Geometry` created manually won't have a renderer passed in, I think we need to handle the case where there is no renderer, and mayube default to the full line behaviour:
```suggestion
      if(!this.renderer?._simpleLines){
```

Reply:
Also, do you think we could intent the block inside this to keep the indentation consistent everywhere?